index.bs

```diff
@@ -2380,6 +2415,7 @@ To <dfn>compute the scopes channel capacity of a source</dfn> given a positive i
 A <dfn>source-registration JSON key</dfn> is one of the following:
 
 <ul dfn-for="source-registration JSON key">
+<li>"<dfn><code>aggregatable_bucket_max_budget</code></dfn>"
```

Comment:
Is the `max` part of this field name really adding anything? The term `budget` already implies that there's a maximum. This might be more readable as `aggregatable_bucket_budgets`.

Reply:
I think adding `max` is to add clarity that this is the maximum budget the bucket can use, but not reserving the budget exclusively for this bucket.

Reply:
+1 to Andrew. The definition of the word, "budget," is already something like "we have this much we can use."

Reply:
Synced offline, we will rename this to `named_budgets`